keps/sig-auth/2799-reduction-of-secret-based-service-account-token/README.md

```diff
@@ -227,10 +227,10 @@ legacy tokens for security practices.
 
 #### Beta -> GA Graduation
 
-- [ ] In use by multiple distributions
-- [ ] Approved by PRR and scalability
-- [ ] Any known bugs fixed
-- [ ] Tests passing
+- [x] In use by multiple distributions
+- [x] Approved by PRR and scalability
```

Comment:
get an ack from PRR reviewer on the tracking portion of the KEP (no changes since beta, so I don't expect any issues)